app/presenters/search_result_presenter.rb

```diff
@@ -8,19 +8,19 @@ class SearchResultPresenter
            :government_name,
            :format,
            :es_score,
-           to: :search_result
+           to: :document
```

Comment:
To me, `search_result` is a clearer name than `document`. Is there a way we can have a document become a `result`?